draft-ietf-quic-tls.md

```diff
@@ -1517,8 +1517,25 @@ TLS_FATAL_ALERT_RECEIVED (0xC000001E):
 
 # IANA Considerations
 
-This document has no IANA actions.  Yet.
-
+This document does not create any new IANA registries, but it does utilize the
+following registries;
+
+* QUIC Transport Parameter Registry {{QUIC-TRANSPORT}} - This registry
+uses the Specificatino Required policy {{!RFC5226}}.  IANA is to register the
```

Comment:
I don't think it's strictly required.  Including it will save the chairs chasing down the registry policy, but maybe that's not a good enough reason to include it.